servant-prometheus.cabal

```diff
@@ -29,7 +29,7 @@ library
                      , Servant.Prometheus.Internal.StatusCodes
   build-depends:       base >= 4.7 && < 4.15
                      , prometheus-client >= 1.0
-                     , servant > 0.10 && < 0.17
+                     , servant
```

Comment:
Why?

Reply:
```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for servant-prometheus-0.2.0.0:
    servant-0.18.3 from stack configuration does not match >0.10 && <0.17  (latest matching
                   version is 0.16.2)
needed since servant-prometheus is a build target.

Some different approaches to resolving this:

  * Set 'allow-newer: true'
    in /home/alexd/.stack/config.yaml to ignore all version constraints and build anyway.

  * Recommended action: try adding the following to your extra-deps
    in /home/alexd/Projects/serokell/servant-prometheus/stack.yaml:

- servant-0.16.2@sha256:6c5d19d09a9dd2f2b2bf99193f944491f8374802678b79ec729e3f3ae51e216d,5867

Plan construction failed
```
I can set constraints to the latest version, but I think that this package uses such `servant` features which probably will not change in the future.

Reply:
Also stack has its own constraints

Reply:
Please, bump the constraint to the current version of servant. That is how versioning is supposed to be done.

I don’t what constraints stack has on its own, but not everyone is using stack.

Reply:
@kirelagin done